docs/QuickStart.md

```diff
@@ -0,0 +1,58 @@
+## Getting Started / Quick Start
+
+### Features
+
+* Parses Mustache templates and an extended template format (in YAML)
+* Generates a view schema from parsed template data
```

Comment:
Use "parameter schema" instead of "view schema" for consistency

Reply:
I've updated this in the README #features.